policyengine_us/parameters/gov/states/hi/tax/income/subtractions.yaml

```diff
@@ -0,0 +1,19 @@
+description: Hawaii counts these subtractions from adjusted gross income.
+values:
+  2021-01-01:
+    - pension_income # Line 13
+    - taxable_social_security # Line 14
+    - hi_military_reserve_or_national_guard_duty_pay_exclusion # Line 15
```

Comment:
recreate this class with just the metadata and no formula from:
https://github.com/PolicyEngine/policyengine-us/pull/3052

Reply:
we will rename to variable as well